middleware/walletValidator.middleware.js

```diff
@@ -159,6 +164,7 @@ const validationCustomMiddleware = (schemaBody, schemaQuery) => (req, res, next)
         else next();
     }
 }
+const cryptoListValidation = validationMiddleware(schemas.cryptoListSchema, 'query');
```

Comment:
I suggest using a validation middleware in the exports object.
cryptoListValidation : validationMiddleware(schemas.cryptoListSchema, 'query');